packages/espresso/android/src/main/java/androidx/test/espresso/flutter/matcher/FlutterMatchers.java

```diff
@@ -99,8 +99,7 @@ public void describeTo(Description description) {
     @SuppressWarnings("deprecation")
```

Comment:
Is this suppressed warning still needed? I think not since you removed the deprecated class.

Reply:
No it isn't, thanks!